lisa/sut_orchestrator/baremetal/cluster/cluster.py

```diff
@@ -0,0 +1,43 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from typing import Any, Type
+
+from lisa import features, schema
+from lisa.environment import Environment
+from lisa.util import InitializableMixin, subclasses
+from lisa.util.logger import get_logger
+
+from ..schema import ClusterSchema
+
+
+class Cluster(subclasses.BaseClassWithRunbookMixin, InitializableMixin):
+    def __init__(
+        self,
+        runbook: ClusterSchema,
+    ) -> None:
+        super().__init__(runbook=runbook)
+        self.cluster_runbook: ClusterSchema = self.runbook
+        self._log = get_logger("cluster", self.__class__.__name__)
+
+    @classmethod
+    def type_schema(cls) -> Type[schema.TypedSchema]:
+        return ClusterSchema
+
+    def deploy(self, environment: Environment) -> Any:
+        raise NotImplementedError()
+
+    def has_serial_console(self) -> bool:
+        raise NotImplementedError()
+
+    def get_serial_console(self) -> features.SerialConsole:
+        raise NotImplementedError()
+
+    def start(self) -> None:
```

Comment:
How about define StartStop feature for each cluster like SerialConsole?

Reply:
you mean define like below way? I didn't implement get_serial_console yet. not sure how to implement in the method. should it return features.StartStop or the baremetal features.StartStop ?
```
    def get_start_stop(self) -> features.StartStop:
        raise NotImplementedError()
```

Reply:
Sorry, I didn't figure out the actual implement for this. 
Can you check the features.py, I think current stratstop in features.py works, if we define `def get_start_stop(self) -> features.StartStop:`, do we still need features.py? when `get_start_stop` will be used? I didn't figure it out.